api/app_analytics/split_testing/views.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from django.http import Http404
+from rest_framework import viewsets
+
+from environments.authentication import EnvironmentKeyAuthentication
+from environments.permissions.permissions import EnvironmentKeyPermissions
+
+from .models import SplitTest
+from .permissions import SplitTestPermissions
+from .serializers import ConversionEventSerializer, SplitTestSerializer
+
+
+class ConversionEventViewSet(viewsets.ModelViewSet):
+    permission_classes = (EnvironmentKeyPermissions,)
+    authentication_classes = (EnvironmentKeyAuthentication,)
+    serializer_class = ConversionEventSerializer
+    http_method_names = ["post"]
```

Comment:
Unless there's a specific reason for doing it this way, I'd rather use the `GenericViewset` / mixins approach. So the class definition for this viewset for example becomes: 

```python
from rest_framework import mixins

...

class ConversionEventViewSet(viewsets.GenericViewSet, mixins.CreateModelMixin):
```

Reply:
Ok cool, I'll default to that pattern for this project when using class based ViewSets.